scripts/spack/specs.json

```diff
@@ -14,26 +14,26 @@
     "__comment__":"##############################################################################",
 
     "toss_3_x86_64_ib":
-    [ "clang@9.0.0~cpp14+devtools+mfem",
-      "clang@10.0.0~cpp14+devtools+mfem",
-      "gcc@8.1.0~cpp14+devtools+mfem+scr",
-      "gcc@8.1_no_fortran~cpp14~fortran+devtools+mfem",
-      "intel@18.0.2~cpp14~openmp+devtools+mfem",
-      "intel@19.0.4~cpp14~openmp+devtools+mfem" ],
+    [ "clang@9.0.0~cpp14+devtools+mfem+c2c",
+      "clang@10.0.0~cpp14+devtools+mfem+c2c",
+      "gcc@8.1.0~cpp14+devtools+mfem+scr+c2c",
+      "gcc@8.1_no_fortran~cpp14~fortran+devtools+mfem+c2c",
+      "intel@18.0.2~cpp14~openmp+devtools+mfem+c2c",
+      "intel@19.0.4~cpp14~openmp+devtools+mfem+c2c" ],
 
     "blueos_3_ppc64le_ib":
-    [ "clang@9.0.0_upstream_xlf~cpp14~openmp+devtools+mfem",
-      "clang@8.0.1_nvcc_xlf~cpp14~openmp+devtools+mfem+cuda cuda_arch=60",
-      "gcc@7.3.1~cpp14+devtools+mfem",
-      "xl@16.1.1_coral~cpp14~openmp+devtools+mfem",
-      "xl@16.1.1_nvcc~cpp14~openmp+devtools+mfem+cuda cuda_arch=60" ],
+    [ "clang@9.0.0_upstream_xlf~cpp14~openmp+devtools+mfem+c2c",
+      "clang@8.0.1_nvcc_xlf~cpp14~openmp+devtools+mfem+cuda+c2c cuda_arch=60",
+      "gcc@7.3.1~cpp14+devtools+mfem+c2c",
+      "xl@16.1.1_coral~cpp14~openmp+devtools+mfem+c2c",
+      "xl@16.1.1_nvcc~cpp14~openmp+devtools+mfem+cuda+c2c cuda_arch=60" ],
 
     "blueos_3_ppc64le_ib_p9":
-    [ "clang@9.0.0_upstream_xlf~cpp14~openmp+devtools+mfem",
-      "clang@8.0.1_nvcc_xlf~cpp14~openmp+devtools+mfem+cuda cuda_arch=70",
-      "gcc@7.3.1~cpp14+devtools+mfem",
-      "xl@16.1.1_coral~cpp14~openmp+devtools+mfem",
-      "xl@16.1.1_nvcc~cpp14~openmp+devtools+mfem+cuda cuda_arch=70" ],
+    [ "clang@9.0.0_upstream_xlf~cpp14~openmp+devtools+mfem+c2c",
+      "clang@8.0.1_nvcc_xlf~cpp14~openmp+devtools+mfem+cuda+c2c cuda_arch=70",
+      "gcc@7.3.1~cpp14+devtools+mfem+c2c",
+      "xl@16.1.1_coral~cpp14~openmp+devtools+mfem+c2c",
+      "xl@16.1.1_nvcc~cpp14~openmp+devtools+mfem+cuda+c2c cuda_arch=70" ],
```

Comment:
What does `+c2c` added to the end of each of these specs mean?

Reply:
It is turning on the variant c2c.  As opposed to `~c2c` turns them off.  This is necessary because the `c2c` variant is off by default.